sos/plugins/ovn_central.py

```diff
@@ -57,20 +76,39 @@ def setup(self):
 
         # Some user-friendly versions of DB output
         cmds = [
+            'ovn-nbctl show',
+            'ovn-sbctl show',
             'ovn-sbctl lflow-list',
             'ovn-nbctl get-ssl',
             'ovn-nbctl get-connection',
             'ovn-sbctl get-ssl',
             'ovn-sbctl get-connection',
         ]
 
+
         schema_dir = '/usr/share/openvswitch'
+        if containerized:
+            for db in ['nb', 'sb']:
+                cmd = ("%s cp %s:/usr/share/openvswitch/ovn-%s.ovsschema "
+                       "/tmp/" % (self._container_runtime,
+                                  self._container_name, db))
+                if self.get_command_output(cmd)['status'] != 0:
```

Comment:
What is the purpose to copy the files? (to get the copy command output that is usefull for some troubleshooting?)

Is it safe to expose that file to publicly available `/tmp` directory - can't it be misused by some intruder?

Reply:
> What is the purpose to copy the files? (to get the copy command output that is usefull for some troubleshooting?)
> 
> Is it safe to expose that file to publicly available `/tmp` directory - can't it be misused by some intruder?
> 
> (I think I got this; you collect the file in add_database_output and I guess it is really safe to expose them since non-containerised environment exposes them in similar way - am I right?)

Yes, you're right. On non containerized environments, this is exposed in the host filesystem while in containerized environments we want to read the files to figure out what tables to pull data from.

Re. security, these files are just the DB schema and are publicly available anyways in the OVS/OVN repositories, they do not contain any data but just the schema definition.